GUI/Controls/ModInfoTabs/Metadata.cs

```diff
@@ -31,22 +31,31 @@ public void UpdateModInfo(GUIMod gui_module)
                 MetadataTable.SuspendLayout();
                 MetadataModuleVersionTextBox.Text = gui_module.LatestVersion.ToString();
                 MetadataModuleLicenseTextBox.Text = string.Join(", ", module.license);
-
+                DownloadCountTextBox.Text = gui_module.DownloadCount.ToString() { Value = $"{mod.DownloadCountTextBox.Text:N0}" };
```

Comment:
`ToString()` doesn't accept a `{}` code block after it, `mod` isn't defined here, `GUIMod` doesn't have a `DownloadCountTextBox` property, and `Text` is the property we're trying to _set_, so we shouldn't reference it.

Try this:

```suggestion
                DownloadCountTextBox.Text = $"{gui_module.DownloadCount:N0}";
```

Reply:
Okay, but doesn't `DownloadCountTextBox.Text` be converted from `int` to `string` since that is what has been the issue before where `ToString()` wasn't there. Or will it be done automatically?

Reply:
The `$"{expression}"` syntax automatically converts the value of `expression` to a string.

Reply:
Awesome!